### PR TITLE
add debugging job

### PR DESCRIPTION
**Pull request by jianlinliu**

### CODE REVIEW

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-8

**Comment by jianlinliu** [comment]
/rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-debug

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-debug

**Comment by jianlinliu** [comment]
@liangxia pls help review.

**Comment by jianlinliu** [comment]
also cc @jhou1 to review.

**Comment by liangxia** [comment]
You wan to merge debug jobs into CI ? 

Is it possible to debug in this PR without merging ? 
If merging is a must, maybe only add the debug job for one version (instead of to every versions) ?

**Comment by jianlinliu** [comment]
> You wan to merge debug jobs into CI ?

Yes. We write a dedicated doc to help other team member who is not familiar with it to follow.  You can treat the debug job as a template job.

> Is it possible to debug in this PR without merging ?
For me, totally okay without merging. But for other team member who is not familiar with it, that would be a bit hard.

> If merging is a must, maybe only add the debug job for one version (instead of to every versions) ?

Different version need different configurations in job configuration level, e.g: payload, executor image version, I want to make the debugging process easier for the team member who is not familiar with it.

**Comment by jianlinliu** [comment]
> > > You wan to merge debug jobs into CI ?
> > 
> > 
> > Yes. We write a dedicated doc to help other team member who is not familiar with it to follow. You can treat the debug job as a template job.
> 
> Does it mean we will have more teammates want to merge debugging PRs into CI as they follow the doc ?

No more, we only have 1 placeholder debugging job per one version, teammates always need to file a new PR to update its content for online debugging, once his/her testing get completed, his/her PR can be closed.

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml [hunk 1]
```diff
@@ -33,6 +37,21 @@ resources:
...
+    - as: online-debug
+      commands: sleep 7200
+      from: upi-installer
+      resources:
+        requests:
+          cpu: 200m
+          memory: 100Mi
+    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
+    pre:
+    - chain: cucushift-installer-rehearse-aws-ipi-provision
```
Comment (liangxia) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml:54]: Several enhance, 
```
- as: installer-rehearse-debug
  cron: '@yearly'
  steps:
    cluster_profile: aws-qe
    env:
      SLEEP_DURATION: 6h
    post:
    - ref: wait
    - chain: cucushift-installer-rehearse-aws-ipi-deprovision
    pre:
    - chain: cucushift-installer-rehearse-aws-ipi-provision
```

Comment (jianlinliu) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml:50]: `wait` ref is using cli image, while `upi-installer` image have more tools installed, that would help the debugging, so I introduced a new step - `cucushift-installer-wait`

Others enhance is done.

**Comment by jianlinliu** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.11-installer-rehearse-debug

**Comment by jianlinliu** [comment]
I did some final minor polish, pls help review again.

**Pull request merged**